common/content/finder.js

```diff
@@ -124,15 +128,27 @@ var Finder = Module("finder", {
 
             // PDF.JS files are different, they use their own messages to communicate the results.
             // So we piggyback the end changes to the findbar when there are any.
-            findbar._original_updateControlState = findbar.updateControlState;
+            findbar._vimpbackup_updateControlState = findbar.updateControlState;
             findbar.updateControlState = function(aResult, aFindPrevious) {
-                this._original_updateControlState(aResult, aFindPrevious);
+                this._vimpbackup_updateControlState(aResult, aFindPrevious);
                 finder.onFindResult({
                     searchString: this._findField.value,
                     result: aResult,
                     findBackwards: aFindPrevious
                 });
             };
+
+            // Normally the findbar appears to notify on failed results.
+            // However, this shouldn't happen when we're finding through the command line,
+            // even though that way we lose any kind of no matches notification until we
+            // stop typing altogether; something to work on at a later time:
+            // - show the quick findbar which will hide after a few seconds?
+            // - or notify the user somehow in the command line itself?
```

Comment:
I didn't do this in this patch at least because I didn't know how to tackle it. The command line's echo methods only really show stuff when not inputting. I didn't want to create new methods/statuses/whatever because I don't really know what I'm doing there. :stuck_out_tongue_winking_eye: